.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
+**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.
+
+## What is the current behavior?
+<!-- Please describe the current behavior and link to a relevant issue. -->
+
+
+## Issue Number:
+
+
+## What is the new behavior?
+
+
+## Does this PR introduce a breaking change?
```

Comment:
this can be removed as it duplicates with my suggested change to add it to the checklist.